### PR TITLE
Steinar/data pipeline

### PR DESCRIPTION
**Pull request by steinarkarlsson**
Main.py now updates MongoDB with the latest API data.

### CODE REVIEW

File: etl/main.py [hunk 1]
```diff
@@ -19,8 +16,15 @@
     p['_id'] = p.pop('id')
 
 # upload to mongodb client (replace <password> with actual password)
-client = pymongo.MongoClient('mongodb+srv://steinar:<password>@fpl-cluster.ygbi5gm.mongodb.net/?retryWrites=true&w=majority')
-db = client.raw
+client = pymongo.MongoClient('mongodb+srv://steinar:steinar@fpl-cluster.ygbi5gm.mongodb.net/?retryWrites=true&w=majority')
```
Comment (James-Leslie) [inline etl/main.py]: Let's try to look into how we can avoid storing our password in the public repo.

Comment (steinarkarlsson) [reply etl/main.py]: I have set up an untracked config file that contains the login information.

File: etl/main.py [hunk 2]
```diff
@@ -18,9 +16,18 @@
 for p in players:
     p['_id'] = p.pop('id')
 
+
+
 # upload to mongodb client (replace <password> with actual password)
-client = pymongo.MongoClient('mongodb+srv://steinar:<password>@fpl-cluster.ygbi5gm.mongodb.net/?retryWrites=true&w=majority')
-db = client.raw
+client = pymongo.MongoClient('mongodb+srv://' + config.username + ':' + config.password + '@fpl-cluster.ygbi5gm.mongodb.net/?retryWrites=true&w=majority')
```
Comment (steinarkarlsson) [inline etl/main.py]: The client now requires config.py to access the username and password. #3

Comment (James-Leslie) [reply etl/main.py]: Yes, this is a nice option.
You could also write what's called an 'f-string', which would look like this:
```python
client = pymongo.MongoClient(f'mongodb+srv://{config.username}:{config.password}@fpl-cluster.ygbi5gm.mongodb.net/?retryWrites=true&w=majority')

**Comment by James-Leslie** [comment]
Closes #3

File: etl/main.py (hunk 2 above)
Comment (steinarkarlsson) [reply etl/main.py]: That's good to know!

**Review by James-Leslie — APPROVED** [review]

**Comment by James-Leslie** [comment]
I have approved this PR, so you may now go and merge into the `data-pipeline` branch

**Pull request merged**